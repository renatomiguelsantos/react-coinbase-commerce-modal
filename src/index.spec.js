import React from "react";
import CoinbaseCommerceButton from "./index";

test("it renders and matches its snapshot", () => {
  const wrapper = shallow(<CoinbaseCommerceButton checkoutId={"aaaa"} />);
  expect(
    wrapper
      .children()
      .first()
      .find("IFrame").length
  ).toEqual(0);
  expect(wrapper).toMatchSnapshot();
});

test("it opens up an iframe when its button is clicked", () => {
  const wrapper = shallow(<CoinbaseCommerceButton checkoutId={"aaaa"} />);
  expect(wrapper).toMatchSnapshot();
});
