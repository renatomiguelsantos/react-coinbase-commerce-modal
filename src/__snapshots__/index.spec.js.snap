// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it opens up an iframe when its button is clicked 1`] = `<div />`;

exports[`it renders and matches its snapshot 1`] = `<div />`;
